internal/cmd/gocdk/main.go

```diff
@@ -161,25 +160,23 @@ func (pctx *processContext) Println(a ...interface{}) {
 	pctx.outlog.Println(a...)
 }
 
-// ModuleRoot searches the given directory and those above it for the Go
-// module root. It also ensures that the module looks like a Go CDK project,
-// based on the existence of a "biomes" directory.
+// ModuleRoot searches the given directory and those above it for the project
+// root, based on the existence of a "go.mod" file and a "biomes" directory.
 func (pctx *processContext) ModuleRoot(ctx context.Context) (string, error) {
-	c := exec.CommandContext(ctx, "go", "list", "-m", "-f", "{{.Dir}}")
-	c.Dir = pctx.workdir
-	output, err := c.Output()
-	if err != nil {
-		return "", xerrors.Errorf("couldn't find a Go module root at or above %s", pctx.workdir)
-	}
-	output = bytes.TrimSuffix(output, []byte("\n"))
-	if len(output) == 0 {
-		return "", xerrors.Errorf("couldn't find a Go module root at or above %s", pctx.workdir)
-	}
-	root := string(output)
-	if _, err := os.Stat(biomesRootDir(root)); err != nil {
-		return "", xerrors.Errorf("Go module root %s doesn't look like a Go CDK project (no biomes/ directory)", root)
+	var prevDir string
```

Comment:
Optional: you could move this declaration to the loop clause:

```go
for dir, prevDir := pctx.workdir, ""; ; dir = filepath.Dir(dir) {
```

Reply:
Done.